R/transformCounts.R

```diff
@@ -28,7 +28,7 @@
 #' A numeric value for scaling. Values are multiplied with specified value.
```

Comment:
scalingfactor -> scalingFactor

Reply:
Oh, thanks. I did not see that

Changed